src/components/layout/nav.js

```diff
@@ -547,9 +547,11 @@ const NavDesktop = ({ base }) => {
     )
 }
 
-export const Nav = ({ base }) => {
+export const Nav = ({ base, CompotentAbove }) => {
     return (
         <NavWrapper>
+            {CompotentAbove !== undefined ? <CompotentAbove /> : ''}
```

Comment:
```suggestion
            {CompotentAbove !== undefined && <CompotentAbove />}
```

Reply:
done. also, changed prototypes to accept bool for avoiding type mismatch error.

Reply:
done.

Reply:
btw just checking for `undefined` will cause an error because of the updated passing condition on responsible-trading/index.js. So checked for both `undefined` and `false`.